raven/contrib/tornado/__init__.py

```diff
@@ -169,9 +169,13 @@ def get_sentry_user_info(self):
         `tornado.web.RequestHandler.get_current_user` tests postitively for on
         Truth calue testing
         """
+        try:
+            user = self.get_current_user()
+        except:
```

Comment:
A good practice in Python is to always "except Exception" when you want to catch all.

That ensures something like SystemExit or KeyboardInterrupt doesn't get caught here